scipy/sparse/csgraph/_shortest_path.pyx

```diff
@@ -430,22 +430,39 @@ def dijkstra(csgraph, directed=True, indices=None,
         will be equal to np.inf (i.e., not connected).
 
         .. versionadded:: 0.14.0
+    min_only: bool, optional
+￼        If False (default), for every node in the graph, find the shortest path to every node in indices.
+        If True, for every node in the graph, find the shortest path to any of the nodes in indices (which
+        can be substantially faster).
+
+        .. versionadded:: 1.3.0
 
     Returns
     -------
-    dist_matrix : ndarray
-        The matrix of distances between graph nodes. dist_matrix[i,j]
-        gives the shortest distance from point i to point j along the graph.
-
-    predecessors : ndarray
+    dist_matrix : ndarray, shape ([n_indices, ]n_nodes,)
+        The matrix of distances between graph nodes. If min_only=False, dist_matrix
+        has shape (n_indices, n_nodes) and dist_matrix[i, j] gives the shortest distance
+        from point i to point j along the graph. If mult_target=True, dist_matrix has
+        shape (n_nodes,) and contains the shortest path from each node to any
+        of the nodes in indices.
+
+    predecessors : ndarray, shape ([n_indices, ]n_nodes,)
+        If min_only=False, this has shape (n_indices, n_nodes),
+        otherwise it has shape (n_nodes,).
         Returned only if return_predecessors == True.
         The matrix of predecessors, which can be used to reconstruct
         the shortest paths.  Row i of the predecessor matrix contains
         information on the shortest paths from point i: each entry
         predecessors[i, j] gives the index of the previous node in the
         path from point i to point j.  If no path exists between point
         i and j, then predecessors[i, j] = -9999
-
+    sources: ndarray, shape(n_nodes)
+        only returned if min_only=True and return_predecessors=True.
```

Comment:
Needs proper capitalization on start of sentences

Reply:
done